src/dashboards/BusStop/DepartureTile/index.tsx

```diff
@@ -72,6 +84,13 @@ const DepartureTile = ({ stopPlaceWithDepartures }: Props): JSX.Element => {
                 <Heading2>{stopPlaceWithDepartures.name}</Heading2>
                 <div className="tile__header__icons">{headerIcons}</div>
             </header>
+            {walkInfo ? (
+                <div className="tile__walking-time">
+                    {`${formatWalkTime(walkInfo.walkTime)} (${Math.ceil(
```

Comment:
Eg trur ikkje walkDistance kan vere desimaltall, så det skal vere trygt å droppe `Math.ceil`. :)

Reply:
Det kom faktisk mange desimaltall, så måtte legge `Math.ceil` for å gjøre det fint.